proto/answer.proto

```diff
@@ -51,7 +51,7 @@ message ConceptMapGroup {
 
 message Numeric {
     oneof value {
-        int64 long_value = 1;
+        sint64 long_value = 1;
```

Comment:
This is worth a comment to describe why we have the change, for future reference.